src/media/local-track.ts

```diff
@@ -34,7 +34,7 @@ export type TrackEffect = any;
 // export type TrackEffect = BaseMicrophoneEffect | BaseCameraEffect;
 
 /**
- * Basic Track class. Wrapper for LocalTrack from 'webrtc-core'.
+ * Local track manages effects which gets extended by video and audio.
```

Comment:
Oops, I must have had a filter on when I saw this line before (I didn't see the updated comment).